### ISSUE TITLE
feat: add method to retrieve the API usage so far in the cycle

### ISSUE THREAD

**Issue by ngupta23** [Nixtla/nixtla#533]
### Description

Add a method to the client to retrieve the API usage in the current cycle.

### Use case

While experimenting, users may be interested in dynamically retrieving their API usage using the SDK to make tradeoffs with experiments and prod usage.

**Comment by ngupta23** [comment]
Per @Yibei990826, the following can be used in the mean time.

```python
import requests

url = "https://api.nixtla.io/usage"

headers = {
  'Authorization': 'Bearer YOUR_API_KEY'
}

response = requests.request("GET", url, headers=headers)

print(response.text)
```

**Comment by jmoralez** [comment]
Fixed by #548
